documentation/sdkautomation/README.md

```diff
@@ -1,22 +1,22 @@
 # SDK Automation Customization
 
-This is the specification of the new SDK Automation customization configuration.
-Old customization that hardcoded in sdk automation will still work but this new
-approach is preferred.
+This is the specification of the SDK Automation customization configuration.
 
 ## SDK Automation workflow
 
-### Opened PR Validation Trigger
+### Spec Pull Request Trigger
 
-SDK Automation is launched with matrix in azure pipeline. For each language configured:
+SDK Automation is launched in azure pipeline. For each language configured:
 
-1. Get the PR merge commit and clone the spec repo on the merge commit.
+1. Clone the spec repo on the merge commit and clone the specified SDK language repo from the main branch.
 
-2. Get the PR changed file list. For each changed file, find the nearest readme.md in parent folder. Get list of related readme.md.
+2. Identify the TypeSpec project or the `readme.md` file based on the PR changed files.
 
-3. Filter which sdk will be generated:
+3. Trigger the `spec-gen-sdk` pipelines for five SDK languages.
```

Comment:
it would depend on the languages specified for the project, no?

Reply:
Answered my question below :)

Reply:
No, it will always trigger five pipelines. Within the pipeline run, it will report the warnings if the specific language is not enabled the SDK configuration.